src/gui/search/searchsortmodel.cpp

```diff
@@ -29,17 +29,89 @@
 #include "searchsortmodel.h"
 
 SearchSortModel::SearchSortModel(QObject *parent)
-    : QSortFilterProxyModel(parent)
+    : base(parent)
+    , m_isNameFilterEnabled(false)
+    , m_minSeeds(0)
+    , m_maxSeeds(-1)
+    , m_minLeechs(0)
+    , m_maxLeechs(-1)
+    , m_minSize(0)
+    , m_maxSize(-1)
 {
 }
 
+void SearchSortModel::enableNameFilter(bool enable)
+{
+    m_isNameFilterEnabled = enable;
+}
+
+void SearchSortModel::setNameFilter(const QString &searchTerm)
+{
+    m_searchTerm = searchTerm;
+    if (searchTerm.length() > 2
+        && searchTerm.at(0) == QLatin1Char('"') && searchTerm.at(searchTerm.length() - 1) == QLatin1Char('"')) {
+        m_searchTermWords.append(m_searchTerm.mid(1, m_searchTerm.length() - 2));
```

Comment:
Yes. Obviously, the list was flooded with the copy of this word :)

On 4 April 2016 at 00:25, sledgehammer999 <notifications@github.com> wrote:

> In src/gui/search/searchsortmodel.cpp
> <https://github.com/qbittorrent/qBittorrent/pull/3989#discussion_r58318785>
> :
>
> >  {
> >  }
> >
> > +void SearchSortModel::enableNameFilter(bool enable)
> > +{
> > +    m_isNameFilterEnabled = enable;
> > +}
> > +
> > +void SearchSortModel::setNameFilter(const QString &searchTerm)
> > +{
> > +    m_searchTerm = searchTerm;
> > +    if (searchTerm.length() > 2
> > +        && searchTerm.at(0) == QLatin1Char('"') && searchTerm.at(searchTerm.length() - 1) == QLatin1Char('"')) {
> > +        m_searchTermWords.append(m_searchTerm.mid(1, m_searchTerm.length() - 2));
>
> Do you mean you should assign and not append? I admit I didn't catch too :S
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/qbittorrent/qBittorrent/pull/3989/files/e5b9855c7860a6ea54383758200bbb7f91116e04#r58318785>
>